pandas/core/strings.py

```diff
@@ -1890,18 +1912,22 @@ def rindex(self, sub, start=0, end=None):
                                docstring=_shared_docs['ismethods'] %
                                _shared_docs['isdecimal'])
 
+    # TODO: Use this instead of wrapping all of these methods individually?
+    def _delegate_method(self, name, *args, **kwargs):
+        # TODO: It would be really nice to keep the signatures
+        method = getattr(self.values, name)
+        res = method(*args, **kwargs)
+        # TODO: Should this get wrapped in an index?
+        return res
```

Comment:
TODO: even though it isn't explicitly needed, having `StringMethods` subclass `PandasDelegate` will clarify its parallels with the other accessors.

Plus actually *using* `_delegate_method` could get rid of a lot of boilerplate.